test/e2e/__snapshots__/basics.test.js.snap

```diff
@@ -17,7 +17,7 @@ Options:
   -h, --help     Show help                                             [boolean]
   -v, --version  Show current version                                  [boolean]
 
-Copyright 2019 Contentful"
+Copyright 2013 Contentful"
```

Comment:
Not 100% sure about 2013 here 🤔

Reply:
When Contentful was founded ;)

Reply:
Only for snapshot testing though, the current year will be printed for users